napari/_qt/_tests/test_qt_viewer.py

```diff
@@ -291,6 +291,27 @@ def test_qt_viewer_data_integrity(make_test_viewer, dtype):
     assert datamean == imean
 
 
+def test_slice_labels(make_test_viewer):
```

Comment:
Moving this wasn't strictly necessary, but it was a test using the whole viewer in qt_dims. confused me up while debugging. could move it back, but hopefully people are ok with move